Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -342,7 +342,7 @@ instance (priority := 200) toModule : Module R A where
 #align algebra.to_module Algebra.toModule
 
 -- porting note: this caused deterministic timeouts later in mathlib3 but not in mathlib 4.
--- attribute [instance 0] Algebra.toSMul
+attribute [instance 0] Algebra.toSMul
```

Comment:
This does the opposite of what your PR title claims it does: it was _already_ an instance, this effectively _removes_ it as an instance.

Reply:
Was confused.